lib/validator.js

```diff
@@ -104,7 +107,14 @@ function executeValidator(ctx, name, attribute, setValue) {
     }
     let callableArgs = [ String(value) ];
     callableArgs = callableArgs.concat(args);
-    if (!validator.apply(ctx, callableArgs)) throw new LeoricValidateError(name, field, msg);
+    let result;
+    try {
+      result = validator.apply(ctx, callableArgs);
+    } catch (e) {
+      e.name = ERR_NAME;
+      throw e;
```

Comment:
An unspoken convention I've been using in leoric is that, `event` is simplified as `e`, and `error` is simplified as `err`, can we follow this convention here?

btw, does this change mean when `validator.apply()` fails, it might return false, but also might throw an error? Sounds a bit weird.

Reply:
build-in validator only returns boolean, my fault